app/controllers/settings/followers_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class Settings::FollowersController < ApplicationController
```

Comment:
I could see this being `Settings::FollowerDomainsController` with a single `index` action for the list, and then something like `Settings::FollowerDomains#update` with the array of domain values instead of `purge`.

Related - how do you feel about a "Block" button next to each domain that would just do them one at a time (and redirect right back to list) -vs- the checkboxes to select a bunch and then submit at once?